### PR TITLE
Resolution for #1137

### LINKED ISSUE #1137: use parse_timeframe provided by ccxt instead of a constant dict
Extracted from the PR below, thanks @creslinux, that's a great find!
https://github.com/freqtrade/freqtrade/pull/1101#discussion_r209022604

CCXT provides `exchange.parse_timeframe`
It may be useful to use CCXT's view of timeframes over our own constants

https://github.com/ccxt/ccxt/blob/d76f474927f92d85734675ce8a86a1f754d4e63d/python/ccxt/base/exchange.py#L1235

### PR DESCRIPTION
**Pull request by hroff-1902**
This PR gets rid of TICKER_INTERVAL_MINUTES dict, uses ccxt's parse_timeframe() instead.

Resolves #1137.

Later, in a separate refactoring PR, a single function can be implemented and used, which returns an arrow object, for example.

### CODE REVIEW

**Comment by hroff-1902** [comment]
I supposed to insert here a commit with cleanup for names of the parameters with timeframe intervals (i.e. `interval`, `tick_interval`, etc, --> `ticker_interval`)... Well, it should better be issued in a separate PR for easier reviewing because of its size,

Ready for review and merge.

**Comment by mishaker** [comment]
@hroff-1902 conflicts ;)

**Comment by hroff-1902** [comment]
> @hroff-1902 conflicts ;)

Hmm... Resolved.

File: freqtrade/misc.py [hunk 1]
```diff
@@ -131,3 +132,15 @@ def deep_merge_dicts(source, destination):
             destination[key] = value
 
     return destination
+
+
+def timeframe_to_seconds(ticker_interval: str) -> int:
+    return Exchange.parse_timeframe(ticker_interval)
```
Comment (xmatthias) [inline freqtrade/misc.py]: The only think i would like to see is a docstring for these explaining that it takes the ticker_interval in the format "5m", "1h" ...

it's none-obvious and i had to look it up in the ccxt sourcecode to make sure ...  

Rest looks good to me :+1:

Comment (hroff-1902) [reply freqtrade/misc.py]: Done.

**Pull request merged**